### PR TITLE
Added initial implementiation of getting data from Azure DevOps

### PR DESCRIPTION
**Pull request by jernejk**

### CODE REVIEW

File: src/AutoScrum.AzureDevOps/AzureDevOpsService.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            _httpClient = httpClient;
+
+            byte[]? byteArray = Encoding.ASCII.GetBytes(_config.UserEmail + ":" + _config.Token);
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", Convert.ToBase64String(byteArray));
+        }
+
+        public async Task<Sprint?> GetCurrentSprint()
+        {
+            HttpResponseMessage? result = await _httpClient.GetAsync(new Uri(_config.OrganizationUrl, $"/DefaultCollection/{_config.Project}/_apis/work/teamsettings/iterations?api-version=6.0&$timeframe=current"));
+            if (result?.IsSuccessStatusCode == true)
```
Comment (william-liebenberg) [inline src/AutoScrum.AzureDevOps/AzureDevOpsService.cs:38]: invert the logic -- return null (early outs)

Comment (william-liebenberg) [reply src/AutoScrum.AzureDevOps/AzureDevOpsService.cs:38]: also reduces nesting :)

File: src/AutoScrum.AzureDevOps/Models/DevOpsWorkItem.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿namespace AutoScrum.AzureDevOps.Models
+{
+    public class DevOpsWorkItem
```
Comment (william-liebenberg) [inline src/AutoScrum.AzureDevOps/Models/DevOpsWorkItem.cs:3]: Consistent naming

Change to `AzureDevOpsWorkItem` :)

Comment (william-liebenberg) [reply src/AutoScrum.AzureDevOps/Models/DevOpsWorkItem.cs:3]: or  just `WorkItem`

**Review by william-liebenberg — CHANGES_REQUESTED** [review]

File: src/AutoScrum.AzureDevOps/Models/DevOpsWorkItem.cs (hunk 2 above)
Comment (jernejk) [reply src/AutoScrum.AzureDevOps/Models/DevOpsWorkItem.cs:3]: Hmm, this becomes a pain managing what is what as it conflicts with WorkItem from Nuget.
Externally it doesn't conflict, so I guess it's ok but not ideal.

Comment (jernejk) [reply src/AutoScrum.AzureDevOps/Models/DevOpsWorkItem.cs:3]: I'll map it as WorkItemModel allias when there is a conflict.

**Review by william-liebenberg — APPROVED** [review]
All good

**Pull request merged**